easybuild/easyconfigs/s/snakemake/snakemake-7.18.1-foss-2020a.eb

```diff
@@ -0,0 +1,96 @@
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2'),
+    ('IPython', '7.15.0', '-Python-3.8.2'),
+    ('poetry', '1.0.9', '-Python-3.8.2'),
+    ('GitPython', '3.1.9', '-Python-3.8.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('wrapt', '1.13.3', {
+        'checksums': ['1fea9cd438686e6682271d36f3481a9f3636195578bab9ca3382e2f5f01fc185'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
+        'checksums': ['5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7'],
+    }),
+    ('ConfigArgParse', '1.5.3', {
+        'checksums': ['1b0b3cbf664ab59dada57123c81eff3d9737e0d11d8cf79e3d6eb10823f1739f'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('toposort', '1.7', {
+        'checksums': ['ddc2182c42912a440511bd7ff5d3e6a1cabc3accbc674a3258c8c41cbfbb2125'],
+    }),
+    ('amply', '0.1.4', {
+        'checksums': ['cb12dcb49d16b168c02be128a1527ecde50211e4bd94af76ff4e67707f5a2d38'],
+    }),
+    ('PuLP', '2.5.1', {
+        'checksums': ['27c2a87a98ea0e9a08c7c46e6df47d6d4e753ad9991fea2901892425d89c99a6'],
+    }),
+    ('smart_open', '5.2.1', {
+        'checksums': ['75abf758717a92a8f53aa96953f0c245c8cedf8e1e4184903db3659b419d4c17'],
+    }),
+    ('connection_pool', '0.0.3', {
+        'checksums': ['bf429e7aef65921c69b4ed48f3d48d3eac1383b05d2df91884705842d974d0dc'],
+    }),
+    ('stopit', '1.1.2', {
+        'checksums': ['f7f39c583fd92027bd9d06127b259aee7a5b7945c1f1fa56263811e1e766996d'],
+    }),
+    ('retry', '0.9.2', {
+        'checksums': ['f8bfa8b99b69c4506d6f5bd3b0aabf77f98cdb17f3c9fc3f5ca820033336fba4'],
+    }),
+    ('dpath', '2.0.6', {
+        'checksums': ['5a1ddae52233fbc8ef81b15fb85073a81126bb43698d3f3a1b6aaf561a46cdc0'],
+    }),
+    ('plac', '1.3.5', {
+        'checksums': ['38bdd864d0450fb748193aa817b9c458a8f5319fbf97b2261151cfc0a5812090'],
+    }),
+    ('poetry-core', '1.0.8', {
+        'checksums': ['951fc7c1f8d710a94cb49019ee3742125039fc659675912ea614ac2aa405b118'],
+	'modulename': 'poetry.core',
+    }),
+    ('yte', '1.5.1', {
+        'checksums': ['6d0b315b78af83276d78f5f67c107c84238f772a76d74f4fc77905b46f3731f5'],
+    }),
+    ('reretry', '0.11.1', {
+        'checksums': ['4ae1840ae9e443822bb70543c485bb9c45d1d009e32bd6809f2a9f2839149f5d'],
+    }),
+    ('MarkupSafe', '2.1.1', {
+        'checksums': ['7f91197cc9e48f989d12e4e6fbc46495c446636dfc81b9ccf50bb0ec74b91d4b'],
+    }),
+    ('Jinja2', '3.1.2', {
+        'checksums': ['31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852'],
+    }),
+    ('PyYAML', '6.0', {
+	'modulename': 'yaml',
+        'checksums': ['68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2'],
+    }),
```

Comment:
is there a strong requirement on `PyYAML > 5.3`? If not, I would add a dependency on `('PyYAML', '5.3')` instead

Reply:
Yes, there is hard dependency on PyYAML >= 6.0.